keyboards/kbdfans/kbd67/mkiirgb/keymaps/spx01/config.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2022 spx01 (@spx01)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* --- custom defines only used in my own code --- */
+
+/* easily switch on debugging features */
+#define CUSTOM_DEBUG false
+
+/* hexrgb mode special keys, these shouldn't be KC_A to KC_F or KC_0 to KC_9 */
+/* keycode that triggers saving the current rgb_static color into eeprom */
+#define HEXRGB_SAVE_KC KC_S
+/* keycode that triggers resetting rgb_static to the color saved in the eeprom */
+#define HEXRGB_RESET_KC KC_R
+
+/* --- keyboard configuration macros --- */
+
+#define USB_POLLING_INTERVAL_MS 2
```

Comment:
heads up, default for this is 1 (1000hz)

Reply:
thanks for the heads up, you're right about that